examples/catalog/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Can you just use the same tsconfig in the root?

Reply:
I was thinking you could remove this file and reference the one in root.

Reply:
Then I'll extends tsconifg.json and overwrite some properties.
Because I don't want to include some properties 😅

Reply:
sounds good!